src/library/dlgtagfetcher.h

```diff
@@ -4,21 +4,24 @@
 #include <QList>
 #include <QTreeWidget>
 
+#include "library/trackmodel.h"
 #include "library/ui_dlgtagfetcher.h"
-#include "track/track.h"
 #include "musicbrainz/tagfetcher.h"
+#include "track/track.h"
 
```

Comment:
Please add some brief documentation of what this class does and how it is used. Start lines with three slashes `///` as discussed on [Zulip](https://mixxx.zulipchat.com/#narrow/stream/109171-development/topic/documenting.20the.20project.20for.20new.20developers)